src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -759,12 +790,17 @@ private void VerifyRanOnBackgroundThread(Times expected)
 
         private void VerifyServiceGetIssues(Times expected)
         {
-            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), null, It.IsAny<CancellationToken>()), expected);
+            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<CancellationToken>()), expected);
         }
 
         private void VerifyServiceGetIssues(Times expected, string sonarQubeProjectKey)
         {
-            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(sonarQubeProjectKey, null, It.IsAny<CancellationToken>()), expected);
+            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(sonarQubeProjectKey, It.IsAny<string>(), It.IsAny<CancellationToken>()), expected);
+        }
+
+        private void VerifyServiceGetIssues(Times expected, string sonarQubeProjectKey, string branch)
```

Comment:
New overload for when we want to check both project key and the branch.

I thought about updating the existing tests to so they checked both the project key and branch, but it would have been quite a bit of work for no real added value.